core/services/ocr2/plugins/ocr2keeper/evmregistry/v20/registry.go

```diff
@@ -201,13 +203,10 @@ func (r *EvmRegistry) Name() string {
 	return r.lggr.Name()
 }
 
-func (r *EvmRegistry) Start(ctx context.Context) error {
+func (r *EvmRegistry) Start(_ context.Context) error {
```

Comment:
/nit This may be a matter of preference, but is it more idiomatic to name the param here and have the caller use `r.ctx` as the argument to `Start`?

Reply:
Start() contexts are only to be used during the synchronous start-up. Not stored or handed off to any goroutines. Since this implementation only launched a background worker, there is nothing to pass this argument to.

Reply:
(We can likely simply this weirdness now that there are more context utils funcs in the std library, but we still always want to block on Close() regardless)